packages/ragbits-core/pyproject.toml

```
@@ -51,6 +51,7 @@ chroma = [
     "chromadb>=0.6.3,<1.0.0",
 ]
 local = [
+    "sentence-transformers==4.0.2",
```

Pinning versions like that makes it hard to install ragbits in projects with big number of dependencies. 
```suggestion
    "sentence-transformers>=4.0.2,<5.0.0",
```